data/releases.yml

```diff
@@ -276,6 +276,15 @@
       icon: S
       description: A lightweight collector/sidecar could be deployed closing to the target monitored system, to collect metrics, traces, and logs.
       source:
+        - version: v0.2.0
+          date: Sep. 17th, 2021
+          downloadLink:
+            - name: src
+              link: https://www.apache.org/dyn/closer.cgi/skywalking/satellite/0.2.0/apache-skywalking-satellite-0.2.0-src.tgz
+            - name: asc
+              link: https://downloads.apache.org/skywalking/satellite/0.2.0/apache-skywalking-satellite-0.2.0-src.tgz.asc
+            - name: sha512
+              link: https://downloads.apache.org/skywalking/satellite/0.2.0/apache-skywalking-satellite-0.2.0-src.tgz.sha512
         - version: v0.1.0
```

Comment:
0.1.0 is not recommended. please remove it from website, and remove it from dist svn. It is kept in archive repo automatically.

Reply:
Removed.

Reply:
The 0.1.0 have been delete in svn path: https://dist.apache.org/repos/dist/release/skywalking/satellite/